### PR TITLE
Remove duplicate plus sign for phone numbers

### LINKED ISSUE #8838: Wrong phone format shown in table
**Scenario:**
1. Log in
2. Go to People tab

**Actual:** Numbers have double plus sign at the beginning (when editing number, only 1 plus sign is visible)
Image
Image

**Expected:** Numbers must have single plus sign

### PR DESCRIPTION
**Pull request by khuddite**
Fixes #8838 

1. Summary
It seems [this PR](https://github.com/twentyhq/twenty/pull/8614) caused the issue. We added a plus sign on front-end when the [callingCode](https://github.com/twentyhq/twenty/blob/c735026f6c4f011e82e80713f1554b5cdb4fda5b/packages/twenty-front/src/modules/ui/field/display/components/PhonesDisplay.tsx#L70) retrieved from back-end already has a plus sign.

2. Solution
@guillim Please let me know if I missed a case where the plus sign is not there for the `callingCode`. If so, I think we should check whether or not `callingCode` has a leading plus sign on front-end before adding it. For now, I just removed the code that appends a plus sign on front-end.

3. Screenshots
localhost_3001_objects_people_view=f4aee583-9d0c-4961-a6e1-fd66bc51dfd5

### CODE REVIEW

**Comment by guillim** [comment]
I will look at it soon

**Comment by FelixMalfait** [comment]
Thanks @khuddite!

**Comment by guillim** [comment]
well spotted @khuddite , I will take on from that moment on. It's a good catch

**Review by Weiko — COMMENTED** [review]
Left a few comments @guillim

File: packages/twenty-front/src/modules/object-record/record-field/meta-types/input/components/PhonesFieldInput.tsx [hunk 1]
```diff
@@ -71,11 +71,10 @@ export const PhonesFieldInput = ({ onCancel }: PhonesFieldInputProps) => {
   const defaultCallingCode =
     stripSimpleQuotesFromString(
       fieldDefinition?.defaultValue?.primaryPhoneCountryCode,
-    ) ?? '+1';
-
+    ) ?? '1';
```
[the remark this answers is not in the document]
Comment (guillim) [reply packages/twenty-front/src/modules/object-record/record-field/meta-types/input/components/PhonesFieldInput.tsx:74]: correct

**Comment by khuddite** [comment]
@guillim Your updates look good to me, thanks!

**Review by Weiko — APPROVED** [review]

**Pull request merged**